test/BUILD.gn

```diff
@@ -642,9 +636,6 @@ if (!is_android) {
     if (decentralized_dns_enabled) {
       deps += [ "//brave/browser/decentralized_dns/test:browser_tests" ]
     }
-    if (!is_android && !is_ios) {
-      deps += [ "//brave/components/ipfs/test:brave_ipfs_browser_tests" ]
```

Comment:
Looks like we specifically exclude some tests for android, please add this into your new BUILD.gn for needed files.

Reply:
I believe just !is_android would be needed since ipfs is not enabled on ios.

Reply:
https://github.com/brave/brave-core/blob/master/test/BUILD.gn#L537 all browsertests disabled